pkgs/top-level/julia-packages.nix

```diff
@@ -0,0 +1,60 @@
+{ pkgs
+, lib
```

Comment:
```suggestion
{ lib
```

Reply:
This was added in case some future package addition will need a non-Julia package.